lib/ace/mode/php_completions.js

```diff
@@ -9346,6 +9356,20 @@ var PhpCompletions = function() {
 
         return [];
     };
+    
+    this.getTagCompletions = function(state, session, pos, prefix) {
+        return [{
+            caption: 'php',
+            value: 'php',
```

Comment:
should `<?php` come before `<?=`

Reply:
<?php is more likely to be used which is why I put it first.